blocks/results-list-block/features/results-list/default.jsx

```diff
@@ -1,407 +1,50 @@
-                  </a>
-                </div>
-              ) : null}
-              { (
-                promoElements.showDescription
-            || promoElements.showDate
-            || promoElements.showByline
-              ) ? (
-                <div className="results-list--description-author-container mobile-order-3">
-                  {promoElements.showDescription && descriptionText && (
-                  <a
-                    href={url}
-                    title={headlineText}
-                  >
-                    <SecondaryFont
-                      as="p"
-                      className="description-text"
-                    >
-                      {descriptionText}
-                    </SecondaryFont>
-                  </a>
-                  )}
-                  { (promoElements.showDate || promoElements.showByline) && (
-                  <div className="results-list--author-date">
-                    { promoElements.showByline
-                    && <Byline content={element} list separator={promoElements.showDate} font="Primary" /> }
-                    { promoElements.showDate && <ArticleDate classNames="story-date" date={displayDate} /> }
-                  </div>
-                  )}
-                </div>
-                ) : null}
-            </div>
-          );
-        })}
-        {
-    !!(contentElements && contentElements.length > 0 && seeMore) && (
-      <div className="see-more">
-        <ReadMoreButton
-          type="button"
-          onClick={() => this.fetchStories(true)}
-          className="btn btn-sm"
-          primaryFont={getThemeStyle(arcSite)['primary-font-family']}
-          primaryColor={getThemeStyle(arcSite)['primary-color']}
-          aria-label={this.phrases.t('results-list-block.see-more-button-aria-label')}
-        >
-          {this.phrases.t('results-list-block.see-more-button')}
-        </ReadMoreButton>
-      </div>
-    )
-  }
-      </div>
-    );
-
-    return (
-      <LazyLoad enabled={this.lazyLoad && !this.isAdmin}>
-        <HeadingSection>
-          <Results />
-        </HeadingSection>
-      </LazyLoad>
-    );
-  }
-}
+import Results from './results';
+
+const ResultsList = ({ customFields }) => {
+  const fusionContext = useFusionContext();
+  const { arcSite, isAdmin } = fusionContext;
+  const { lazyLoad } = customFields;
+  const fusionProperties = getProperties(arcSite) || {};
+  const imageProperties = {
+    smallWidth: 158,
+    smallHeight: 89,
+    mediumWidth: 274,
+    mediumHeight: 154,
+    largeWidth: 274,
+    largeHeight: 154,
+    primaryLogoAlt: fusionProperties.primaryLogoAlt,
+    breakpoints: fusionProperties.breakpoints,
+    resizerURL: fusionProperties.resizerURL,
+  };
+
+  return (
+    <LazyLoad enabled={lazyLoad && !isAdmin}>
+      <HeadingSection>
+        <Results
+          customFields={customFields}
+          fusionContext={fusionContext}
+          fusionProperties={fusionProperties}
```

Comment:
Same as above, we should only pass down what is needed, this object isn't as big as the content by could become very large for some clients

Reply:
resolving